semgrep-core/tests/tainting_rules/scala/return_expr.scala

```diff
@@ -0,0 +1,9 @@
+def no_ret(x: Int) : Int = {
```

Comment:
No need to change, but just FYI `tests/tainting_rules` is a leftover from when `semgrep-core -rules` could not parse taint rules and we had a different switch for running them. Nowadays I try to put taint tests under `tests/OTHER/rules/` and I just prefix them with `taint_`.